README.md

```diff
@@ -99,25 +99,23 @@ instead of being passed into the constructor.
 ### Example Usage
 
 ```js
-var Filter = require('broccoli-persistent-filter');
-
-Awk.prototype = Object.create(Filter.prototype);
-Awk.prototype.constructor = Awk;
-function Awk(inputNode, search, replace, options) {
-  options = options || {};
-  Filter.call(this, inputNode, {
-    annotation: options.annotation
-  });
-  this.search = search;
-  this.replace = replace;
+const Filter = require('broccoli-persistent-filter');
+
+class Awk extends Filter {
+  constructor(inputNode, search, replace, options) {
+    options = options || {};
+    super(inputNode, {
+      annotation: options.annotation
+    });
+    this.search = search;
+    this.replace = replace;
+    this.extensions = ['txt'];
+    this.targetExtension = 'txt';
+  }
+  processString(content, relativePath) {
+    return content.replace(this.search, this.replace);
+  };
```

Comment:
I believe this `;` would be illegal syntax when following a function declaration in a `class`.